src/elements/content-preview/ContentPreview.js

```diff
@@ -1167,6 +1179,23 @@ class ContentPreview extends React.PureComponent<Props, State> {
         }
     };
 
+    handleAnnotationsFetchError = () => {
```

Comment:
We've tried to encapsulate annotations logic to `withAnnotations` HOC (apart from setting up the callbacks to Preview SDK which we can't avoid currently -- is this something we could move there as well?